main.py

```diff
@@ -1,51 +1,19 @@
+import Telecom_Cable_Color_Coding
 
-MAJOR_COLORS = ['White', 'Red', 'Black', 'Yellow', 'Violet']
-MINOR_COLORS = ["Blue", "Orange", "Green", "Brown", "Slate"]
-
-
-def color_pair_to_string(major_color, minor_color):
-  return f'{major_color} {minor_color}'
-
-
-def get_color_from_pair_number(pair_number):
-  zero_based_pair_number = pair_number - 1
-  major_index = zero_based_pair_number // len(MINOR_COLORS)
-  if major_index >= len(MAJOR_COLORS):
-    raise Exception('Major index out of range')
-  minor_index = zero_based_pair_number % len(MINOR_COLORS)
-  if minor_index >= len(MINOR_COLORS):
-    raise Exception('Minor index out of range')
-  return MAJOR_COLORS[major_index], MINOR_COLORS[minor_index]
-
-
-def get_pair_number_from_color(major_color, minor_color):
-  try:
-    major_index = MAJOR_COLORS.index(major_color)
-  except ValueError:
-    raise Exception('Major index out of range')
-  try:
-    minor_index = MINOR_COLORS.index(minor_color)
-  except ValueError:
-    raise Exception('Minor index out of range')
-  return major_index * len(MINOR_COLORS) + minor_index + 1
-
-
-def test_number_to_pair(pair_number,
-                        expected_major_color, expected_minor_color):
-  major_color, minor_color = get_color_from_pair_number(pair_number)
+def test_number_to_pair(pair_number,expected_major_color, expected_minor_color):
+  major_color, minor_color = Telecom_Cable_Color_Coding.get_color_from_pair_number(pair_number)
   assert(major_color == expected_major_color)
   assert(minor_color == expected_minor_color)
 
-
 def test_pair_to_number(major_color, minor_color, expected_pair_number):
-  pair_number = get_pair_number_from_color(major_color, minor_color)
+  pair_number = Telecom_Cable_Color_Coding.get_pair_number_from_color(major_color, minor_color)
   assert(pair_number == expected_pair_number)
 
-
 if __name__ == '__main__':
   test_number_to_pair(4, 'White', 'Brown')
```

Comment:
These tests are already part of Telecom_Cable_Color_Coding_Test.py
Is there any value add by keeping similar code here? or am i missing something!